content/docs/1.5/scalers/rabbitmq-queue.md

```diff
@@ -49,15 +58,33 @@ spec:
   triggers:
   - type: rabbitmq
     metadata:
-      # Required
-      host: RabbitMqHost # references a value of format amqp://guest:password@localhost:5672/vhost
       queueName: testqueue
       queueLength: "20"
+    authenticationRef:
+      name: keda-trigger-auth-rabbitmq-conn
+---
+apiVersion: keda.k8s.io/v1alpha1
```

Comment:
Let's create this before the scaledobject for logical ordering

Reply:
Done